src/core/lib/surface/call.cc

```diff
@@ -561,16 +510,14 @@ static void destroy_call(void* call, grpc_error* error) {
     GRPC_CQ_INTERNAL_UNREF(c->cq, "bind");
   }
 
-  get_final_status(c, set_status_value_directly, &c->final_info.final_status,
-                   nullptr, &(c->final_info.error_string));
+  grpc_slice slice = grpc_empty_slice();
```

Comment:
Is this needed? I don't see it being used. Could we just pass nullptr to grpc_error_get_status?